### ISSUE TITLE
Does Jilt support generic classes?

### ISSUE THREAD

**Issue by ThibaultUrien** [skinny85/jilt#5]
It is not mentioned in the documentation.
On my side it seem that classes with generic argument are given a builder that do use the argument but never declare it.

**Comment by skinny85** [comment]
Hey @ThibaultUrien,

thanks for opening the issue!

Can you show the code for the class you're trying to generate the Builder for?

Thanks,
Adam

**Comment by ThibaultUrien** [comment]
Hello Adam, thank for the quick answer.
I don't have the code anymore, but I'll try to come-up with a new one.

**Comment by skinny85** [comment]
Thanks, that will be super helpful in trying to reproduce this issue!

**Comment by skinny85** [comment]
@ThibaultUrien any updates here? 🙂

**Comment by gbaso** [comment]
Not OP, but I'd imagine something like this would be nice:

```java
class MyClass<T> {
  T myField;
}
```

```java
MyClass<String> mc = MyClassBuilder.<String>myclass()
  .myField("some string")
  .build();
```

**Comment by skinny85** [comment]
Thanks @gbaso. Confirming that placing `@Builder` on your example `MyClass` does not work currently.

I'll work on a fix for this.

**Comment by skinny85** [comment]
This has been fixed on the current `develop` branch, and will be included in the next `1.4` release (which will also cover #10, #11 and #12).

**Comment by skinny85** [comment]
This has been fixed in the latest Jilt release (`1.4`).

I'm closing this issue as "Resolved", please comment if you run into any more problems related to this area, and I'll reopen the issue.
